frame/nomination-pools/src/lib.rs

```diff
@@ -2115,6 +2397,101 @@ pub mod pallet {
 			ensure!(bonded_pool.can_nominate(&who), Error::<T>::NotNominator);
 			T::Staking::chill(&bonded_pool.bonded_account())
 		}
+
+		/// Set the commission of a pool.
+		///
+		/// The dispatch origin of this call must be signed by the `root` role of
+		/// the pool. If the pool has a max commission set, the commission supplied
+		/// must be less or equal to that value.
+		///
+		/// If the max commission has not yet been set, then the commission range is
+		/// not bounded. A `payee` must be provided if commission has not yet been
+		/// set (still `None`). Once commission has been set, the `payee` can be
+		/// omitted in further calls. If a `payee` update is desired, the commission
+		/// must still be passed into the call.
+		#[pallet::call_index(14)]
+		#[pallet::weight(T::WeightInfo::set_commission())]
+		pub fn set_commission(
```

Comment:
This does not seem to allow you to reset your commission back to `None`? 

(and I know we discussed this probably before, but I think a `commission: Option<(Perbill, AccountId)>)` that is set-is, without any additional logic might be simpler)

Reply:
Ah yes. I have now simplified this to what you suggested. We can now update `current` to `None`, which also is not throttled - I don't think we would want to throttle removal of commission :)